README.md

```diff
@@ -61,6 +61,54 @@ Depois basta aplicar as `migrations`:
 python manage.py migrate
 ```
 
+* Docker (alpha)
+
+Se você não quiser instalar todas as dependencias, você pode [instalar](https://docs.docker.com/install/) e usar o Docker.
+Para isso, você precisa criar um `.env` como abaixo:
+
+```
+KEEP_FILES=True
+SPIDERMON_TELEGRAM_FAKE=True
+SPIDERMON_SENTRY_FAKE=True
+DJANGO_SETTINGS_MODULE=core.settings
+DJANGO_CONFIGURATION=Prod
+DJANGO_SECRET_KEY=dont-tell-anybody
+```
+
+Depois (somente na primeira vez):
```

Comment:
Pode adicionar instruções sobre a criação do banco também, porfa? :pray:

Reply:
Oi Ana, então, eu to tentando estudar como fazer a criação do banco, e olhando o [Jarbas](https://github.com/okfn-brasil/serenata-de-amor/blob/master/jarbas/README.md#create-and-seed-the-database-with-sample-data) quando rodamos `make migrate` supostamente ele deveria criar o banco também... no momento estou tentando entender por que não está criando.

Reply:
Como está a connection string para o banco no .env? Usando o .env default ele deve criar um banco SQLite